test/asyncJest.spec.js

```diff
@@ -11,12 +11,20 @@ ATENÇÃO!!! Edite apenas este arquivo. Não altere os arquivos da pasta 'src'.
 */
 
 describe("o retorno do telefonema", () => {
-  test("atende", () => {
-    assert.fail();
+  test("atende", async () => {
+    // assert.fail();
     // Insira seu teste assíncrono aqui
+    const call = await answerPhone(true);
+    expect.assertions(1);
+    expect(call).toBe('Oi!');
   });
-  test("ocupado", () => {
-    assert.fail();
+  test("ocupado", async () => {
+    // assert.fail();
     // Insira seu teste assíncrono aqui
+    try {
```

Comment:
```suggestion
    expect.assertion(1);
    try {
```
nesse caso é necessário pois em caso de o teste passar sem cair no catch ele apontara um falso-positivo no teste

Reply:
Grato! nao lembrava dessa.